packages/@okta/vuepress-site/docs/reference/api/authn/index.md

```diff
@@ -872,15 +872,17 @@ Content-Type: application/json
+* If you have the security behavior detection feature enabled and you have a new device behavior configured in a policy rule, a new device is detected if the device fingerprint sent in the `X-Device-Fingerprint` header isn't associated with a previously successful user sign in. For more information about this feature, see [Security Behavior Detection](https://help.okta.com/en/prod/Content/Topics/Security/proc-security-behavior-detection.htm).
+
+> **Note:** Okta is improving its new device behavior detection such that it will no longer use a device fingerprint. This change will be behind a self-service feature flag TODO ADD FRIENDLY NAME HERE available June 15th, 2020. Customers relying on device fingerprinting for new device behavior detection should start sending a `deviceToken` in the [context object](#context-object) to uniquely identify devices for behavior detection. Starting September 1, 2020 the update will be made for all orgs and the device fingerprint will no longer be accepted as a valid device identifier for new device behavior detection. Other features relying on device fingerprinting will not be affected at this time. For more information, see [Security Behavior Detection](https://help.okta.com/en/prod/Content/Topics/Security/proc-security-behavior-detection.htm).
 
 Specifying your own device fingerprint in the `X-Device-Fingerprint` header is a highly privileged operation that is limited to trusted web applications and requires making authentication requests with a valid API token. You should send the device fingerprint only if the trusted app has a computed fingerprint for the end user's client.
 
 > **Note:** The `X-Device-Fingerprint` header is different from the device token. Device-based MFA in the Okta Sign-On policy rules depends on the device token only and not on the `X-Device-Fingerprint` header. To read more about the device token, see [Context Object](#context-object). Device-based MFA would work only if you pass the device token in the [client request context](/docs/reference/api-overview/#client-request-context).
 
-##### Device Fingerprint Best Practices
+##### Best Practices for Generating a Device Fingerprint or Device Token
 
-Use the following recommendations as guidelines for generating and storing a device fingerprint in the `X-Device-Fingerprint` header for both web and native applications.
+Use the following recommendations as guidelines for generating and storing a device fingerprint in the `X-Device-Fingerprint` header or a `deviceToken` in the [context object](#context-object) for both web and native applications.
```

Comment:
i wasn't sure if we wanted to make these changes or not, but thought it made sense since the comm that will be sent out links to this section. I'm not sure if this conflicts with the existing note for device tokens that is higher up on this page:

<img width="897" alt="Screen Shot 2020-05-14 at 3 06 08 PM" src="https://user-images.githubusercontent.com/21957865/81975043-76729100-95f4-11ea-8d57-8aabb2107466.png">

Reply:
Also, this is under the device fingerprint section so not sure it is appropriate to include this here or if we should make a separate section under device token section

Reply:
May be, we could add this in a separate section where we have the existing recommendation for `deviceToken`. `It is recommended that you generate a UUID or GUID for each client and persist the deviceToken as a persistent cookie or HTML5 localStorage item scoped to your web application's origin.`

Something like `Best Practices For Generating A Device Token` section under the  https://developer.okta.com/docs/reference/api/authn/#context-object section?

Also, in https://developer.okta.com/docs/reference/api/authn/#context-object we may want to describe that we will start using the `deviceToken` for behavior detection too if the self service FF is enabeld.

Reply:
makes sense to me. I wasn't sure if we wanted all this in on place or separate places or if these practices even applied to the DT